### ISSUE TITLE
What's the options when training on LSUN dataset?

### ISSUE THREAD

**Issue by YYZ1021** [Newbeeer/pfgmpp#10]
Hi, I just want to know what's the options when training on a 256*256 dataset?

Here is my options:

--outdir=C:\Users\ENeS\Desktop\outputs
--name=train_14
--data=C:\Users\ENeS\Desktop\Dataset\1\FFHQ_256\ffhq-train-256x256.zip
--cond=0
--arch=ddpmpp
--pfgmpp=1
--batch=1
--aug_dim=2048
--duration=50
--lr=1e-3
--tick=10
--dump=1
--cres=1,4,4,4
--lsun=True

I just use your dataset_tool.py to convert the ffhq dataset to 256*256 resolution, and use it as the dataset. Considering my GPU so I changed some training details to accelerate the training speed. 

However, the generated images are not well (I would like to say they are painted by Picasso, but not by the AI, and even worse). I didn't finish the full training, but I used the converged checkpoint (It is also stranged that the loss can be down to 0.01). 

Thank you very much!

**Comment by Newbeeer** [comment]
Hi,

We do implement LSUN Churches 256x256 recently. I will upload the full instructions / pre-trained models recently. On this dataset, I use the following command line for PFGM++:

```
torchrun --standalone --nproc_per_node=8 train.py --outdir=training-runs --name exp_name \
    --data /scratch/ylxu/lsun-church-256.zip --cond=0 --arch=ddpmpp --batch=64 --cres=1,1,2,2,2,2,2 --lr=2e-4 --dropout=0.0 --augment=0.0 --nblock=2 --lsun=1 --pfgmpp=1 --aug_dim D --dump 10

D: recommended : 196608 / 131072 / 8192
exp_name: path to save models

```

Note that $D$ should scale with the data dimension $N$ (196608 in this case).

Best,
Yilun

**Comment by YYZ1021** [comment]
Thank you very much! I will try it now.
